Connection/RESTConnector.cs

```diff
@@ -488,17 +488,22 @@ private IEnumerator ProcessRequestQueue()
                         {
                             Log.Error("RESTConnector.ProcessRequestQueue()", "Exception when initializing WWWForm: {0}", e.ToString());
                         }
+
                         if (req.HttpMethod == UnityWebRequest.kHttpVerbPUT) 
                         {
                             unityWebRequest = UnityWebRequest.Post(url, form);   
                             unityWebRequest.method = "PUT";
                         }
-                        else
+                        else if(req.HttpMethod == UnityWebRequest.kHttpVerbPOST)
```

Comment:
should there be a space before the parens?

Reply:
yes sir!